pydy/viz/tests/test_scene.py

```diff
@@ -59,8 +59,7 @@ def setup(self):
         self.origin = origin
         self.viz_frame = VisualizationFrame(ceiling, block, sphere)
         self.viz_frame_sym_shape = VisualizationFrame(ceiling, block,
-                                                      Sphere(radius=mass /
-                                                             10.0))
+                                                      Sphere(radius = 10.0))
```

Comment:
These tests are correct. We allow symbols to be passed in as the attributes for shapes so that you can define them symbolically. Please don't change the current tests unless you have very good reason to.